.circleci/config.yml

```diff
@@ -124,6 +131,11 @@ jobs:
             TEST_PATH: "tests/unit"
             COVERAGE_ARGS: "-p"
           command: |
+            if [ -f aws_credentials.env ]; then
+              source aws_credentials.env
+            else
+              echo "aws_credentials.env file not found. Skipping setup of non-default aws credentials."
+            fi
```

Comment:
nit: These sections are a bit repetitive. I think it would be great if we could have this in a command, similar to the `prepare-pytest-tinybird` command. You can also implement the conditional quite nicely.
Maybe something like `prepare-account-region-randomization`?

Reply:
If you run this in a command, you could maybe just put the whole `generate-random-aws-credentials` in there and not have a specific command alltogether.
This would simplify the code, have all the account-id and region randomization code together, and it could be "plugged" into a job by just adding the command. However, this would mean that each job would have a different set of randomly generated IDs.

Reply:
Yes I wanted to avoid different set of randomly generated IDs for every job, likewise we have it for default credentials in community tests right now. But yes we could do this as well.